### PR TITLE
fix to remove visibility links on access labels

### PR DESCRIPTION
**Pull request by gordonleacock**
contains:
fix to remove visibility links on access labels in
	modified:   app/helpers/file_set_helper.rb
	modified:   app/views/collections/_show_document_list_row.html.erb
	modified:   app/views/my/_index_partials/_list_works.html.erb

changes for workstations development (to be ignored)

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Application < Rails::Application
     end
 
     # Deploy to /data instead of /
-    config.relative_url_root = '/data'
+    # config.relative_url_root = '/data'
```
Comment (grosscol) [inline config/application.rb]: Why is this added?

**Comment by grosscol** [comment]
All the sample.yml config files need to remain in the repository.

File: app/helpers/file_set_helper.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  def display_multiple(value)
+    return if value.nil?
+    auto_link(html_escape(value.join(" | ")))
+  end
+
+  private
+
+  def render_visibility_badge(presenter)
```
Comment (grosscol) [inline app/helpers/file_set_helper.rb]: Can this be overridden without copy pasting the entire module?

File: app/helpers/file_set_helper.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    # if can? :edit, presenter.solr_document
+    #   render_visibility_link presenter.solr_document
+    # else
```
Comment (grosscol) [inline app/helpers/file_set_helper.rb]: Is this code being commented out to remove it? or is it being left as a placeholder for some future work?

Comment (gordonleacock) [inline app/helpers/file_set_helper.rb]: I will remove it.

File: app/helpers/file_set_helper.rb (hunk 2 above)
Comment (gordonleacock) [inline app/helpers/file_set_helper.rb]: @grosscol: I wasn't sure how to do that. Do you have a suggestion?
